packages/bierzo-wallet/src/routes/payment/index.tsx

```diff
@@ -44,38 +44,37 @@ const onSubmit = (): void => {};
 const Payment = (): JSX.Element => {
   const classes = useStyles();
 
-  const { handleSubmit, invalid, pristine, submitting } = useForm({
+  const { form, handleSubmit, invalid, pristine, submitting } = useForm({
     onSubmit,
```

Comment:
Just FYI, here if you need to validate something at form level, let's say the user can send this payment, or whatever which does not fit in local field validation, it can be specified here. And this function will be validated if the local validation of fields passes.

Good job.